src/index.js

```diff
@@ -22,6 +22,10 @@ function lookupFlowDir(context) {
     : root;
 }
 
+function stopOnExit(context) {
+  return !!(context.settings && context.settings.stopOnExit);
```

Comment:
I think that if I add annotations, this method should return a boolean. So using `!!` would make sense to me.